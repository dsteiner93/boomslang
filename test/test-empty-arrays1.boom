def myfunc(int[] x):
	println("this works now")

myfunc([])

def myfunc(int[][] x):
	println("this also works")

myfunc([[], [], [], []])
myfunc([[1,2], [1,2], [1,2]    ])

int[][] arr1 = default int[0][0]
int[][] arr2 = [[]]
println(arr1)
println(arr2)
