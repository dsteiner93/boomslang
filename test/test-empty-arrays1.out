this works now
this also works
this also works
Array
Array
